src/groups/GroupCard.test.tsx

```diff
@@ -0,0 +1,106 @@
+import { fireEvent, waitFor } from '@testing-library/react';
+import { writeText } from 'clipboard-polyfill/text';
+import React from 'react';
+import { GroupCard } from 'src/groups/GroupCard';
+import { CurrentUserGroupMembership } from 'src/libs/ajax/Groups';
+import { asMockedFn, renderWithAppContexts as render } from 'src/testing/test-utils';
+
+jest.mock('src/libs/nav', (): typeof import('src/libs/nav') => ({
+  ...jest.requireActual('src/libs/nav'),
+  getLink: jest.fn((link) => link),
+}));
+
+jest.mock('clipboard-polyfill/text', () => ({
+  ...jest.requireActual('clipboard-polyfill/text'),
+  writeText: jest.fn().mockResolvedValue(true),
+}));
+
+describe('GroupCard', () => {
+  const testGroup1: CurrentUserGroupMembership = {
```

Comment:
More meaningful names would make the testcases easier to read. For instance-- `memberGroup` and `adminGroup`.